packages/examples/component-library/package.json

```diff
@@ -29,6 +29,7 @@
     "@pixeloven/cli": "^7.0.0-alpha.0",
     "@pixeloven/cli-addon-storybook": "^7.0.0-alpha.1",
     "@pixeloven/cli-core": "^7.0.0-alpha.0",
+    "@storybook/addon-docs": "6.2.7",
```

Comment:
@jimmynotjim This was one of the reason I had the common library so we can wrap dependencies like this.